luigi2.h

```diff
@@ -3094,12 +3097,44 @@ int _UITableMessage(UIElement *element, UIMessage message, int di, void *dp) {
 				}
 			}
 		}
+		UIElementRefresh(element);
```

Comment:
What is this for? Doesn't this cause the table to redraw itself every event?

Reply:
I guess this is no longer needed, i fixed this problem this line solved before i'll just remove it.